types/mixpanel/index.d.ts

```diff
@@ -191,9 +191,9 @@ interface Mixpanel {
      *     });
      *
      * @param properties An associative array of properties to store about the user
-     * @param days How many days since the user's last visit to store the super properties
+     * @param daysOrOptions How many days since the user's last visit to store the super properties or the options object which controls persistence
      */
-    register(properties: { [index: string]: any }, days?: number): void;
+    register(properties: { [index: string]: any }, daysOrOptions?: number | { persistent: boolean }): void;
```

Comment:
Is persistent suppose to be optional ? like in browser? 
Also why not define the interface and use it

Reply:
1. Changed it to be options
2. I avoided creating this interface for the same reason I mentioned in the first comment. This file doesn't include interfaces for the method's parameter types (you can see the `properties` parameter in this function, in general `{ [index: string]: any }` repeats itself multiple times). Do you think that we should add this interface at the top of this file regardless?